libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt

```diff
@@ -797,7 +800,12 @@ class MapboxNavigationViewportDataSource(
                 overviewPitchProperty.get()
             )
         } else {
-            mapboxMap.getCameraOptions()
+            mapboxMap.cameraForCoordinates(
+                listOf(mapboxMap.cameraState.center),
+                mapboxMap.cameraState.padding,
+                mapboxMap.cameraState.bearing,
+                mapboxMap.cameraState.pitch
+            )
```

Comment:
```suggestion
            mapboxMap.cameraState.toCameraOptions
```
or something like that.

Reply:
I pushed a commit to adjust this. The result is the same, but this version is clearer when looking at the codebase.